.gitmodules

```diff
@@ -68,4 +68,3 @@
 [submodule "3rdparty/dasharo-blobs"]
 	path = 3rdparty/dasharo-blobs
 	url = https://github.com/Dasharo/dasharo-blobs.git
-
```

Comment:
@mkopec nit: probably not intended to be modified

Reply:
Yeah, leftover from attempts to build EC here

Reply:
Removed https://github.com/Dasharo/coreboot/pull/481/commits/4e7611536c2168e6ce8aff506f09895064f7506e